var/spack/repos/spack_repo/builtin/packages/py_psyclone/package.py

```diff
@@ -46,7 +46,6 @@ class PyPsyclone(PythonPackage):
     depends_on("py-graphviz", type=("build", "run"))
     depends_on("py-configparser", type=("build", "run"))
     depends_on("py-jinja2", type="build")
-    depends_on("py-jsonschema", type=("build", "run"), when="@2.5.0:")
```

Comment:
FWIW. It is listed in `install_requires` for `2.5.0` (https://github.com/stfc/PSyclone/commit/dadffcae1be526d7b918f100726f8ce20e520aee#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L164 and https://github.com/stfc/PSyclone/blob/dadffcae1be526d7b918f100726f8ce20e520aee/setup.py#L165).

Reply:
Thanks for the quick review.

The dependency has also been removed from the setup.py upstream, see https://github.com/stfc/PSyclone/pull/2980 Do you still want the comment here?

It was also not needed for the last few releases but we forgot to remove it from setup.py in the psyclone repository. I tested from 2.5 upwards and all of them work without it (not surprising because it was only needed for a jupyter tutorial that we deprecated some time ago and is not even part of what Spack installs).

The py-fparser minimum version is set in the "current dependencies" section. Do you want me to bring it down with the other py-fparser depends_on directives or you mean something different?

Reply:
🤦 Missed it in the `current dependencies` section.